src/test/java/nurseybook/model/person/PhoneTest.java

```diff
@@ -24,10 +24,11 @@ public void isValidPhone() {
         assertFalse(Phone.isValidPhone("phone")); // non-numeric
         assertFalse(Phone.isValidPhone("9011p041")); // alphabets within digits
         assertFalse(Phone.isValidPhone("9312 1534")); // spaces within digits
+        assertFalse(Phone.isValidPhone("931201534")); // 7 digits
+        assertFalse(Phone.isValidPhone("9312534")); // 8 digits
```

Comment:
This line should have the comment // 7 digits, while the line above (in line 27) should have the comment // 9 digits.